jekyll/_cci2/authorize-google-cloud-sdk.md

```diff
@@ -4,62 +4,66 @@ title: Authorize Google Cloud SDK
-{: #installing-the-google-cloud-sdk }
+## 1. Install the Google Cloud SDK
+{: #install-the-google-cloud-sdk }
 
-If Debian is an acceptable operating system for your primary container, consider using Google's base Docker image.
-You can find this image on DockerHub as [`google/cloud-sdk`](https://hub.docker.com/r/google/cloud-sdk/).
+If Debian is an acceptable operating system for your primary container, consider using Google's base Docker image. You can find this image on DockerHub as [`google/cloud-sdk`](https://hub.docker.com/r/google/cloud-sdk/).
 
 Otherwise, follow the [Google Cloud SDK installation instructions](https://cloud.google.com/sdk/) for your base image's operating system.
 
-### Creating and storing a service account
-{: #creating-and-storing-a-service-account }
+## 2. Create and store a service account
+{: #create-and-store-a-service-account }
 
 Before you can use any tools in the Google Cloud SDK, you must authorize `gcloud`. Google offers two types of authorization: user accounts and service accounts. Because you are installing the Cloud SDK on CircleCI, the service account is the appropriate choice.
 
-1. Create a service account by following Steps 1-3 of [Google's instructions](https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account). Remember to download the JSON-formatted key file.
+### 1. Create a service account
+{: #create-a-service-account }
+
+Create a service account by following Steps 1-3 of [Google's instructions](https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account). Remember to download the JSON-formatted key file.
+
+### 2. Add a key file
+{: #add-a-key-file }
+
+Add the key file to CircleCI as a [project environment variable](/docs/set-environment-variable/#set-an-environment-variable-in-a-project). In this example, the variable is named `GCLOUD_SERVICE_KEY`. Using this particular name is not required, but it will be used throughout the examples in this document.
+
+### 3. Add environment variables
+{: #add-env-vars }
 
-2. Add the key file to CircleCI as a [project environment variable]({{ site.baseurl }}/set-environment-variable/#set-an-environment-variable-in-a-project). In this example, the variable is named `GCLOUD_SERVICE_KEY`. Using this particular name is not required, but it will be used throughout the examples in this document.
+For convenience, add two more environment variables to your CircleCI project:
 
-3. For convenience, add two more environment variables to your CircleCI project:
-    - `GOOGLE_PROJECT_ID`: the ID of your GCP project.
-    - `GOOGLE_COMPUTE_ZONE`: the default [compute zone](https://cloud.google.com/compute/docs/regions-zones/).
+  - `GOOGLE_PROJECT_ID`: the ID of your GCP project.
+  - `GOOGLE_COMPUTE_ZONE`: the default [compute zone](https://cloud.google.com/compute/docs/regions-zones/).
 
-#### Adding granular permissions
-{: #adding-granular-permissions }
+#### Add granular permissions
```

Comment:
As this bit is kind of an overall note maybe this section could go at the start of the page somewhere, maybe even as a note banner?